Apps/Sandcastle/gallery/Resolution Scaling.html

```diff
@@ -43,6 +43,15 @@
 function startup(Cesium) {
     'use strict';
 //Sandcastle_Begin
+// When browser recommended resolution is enabled, the viewer renders at
+// CSS pixel resolution (96 dppi). Otherwise, the viewer renders at the
+// screen's native resolution.
+
+// Resolution scale applies additional scaling on top.
+
+// Together the final pixel ratio is:
+// resolutionScale * (useBrowserRecommendedResolution ? 1 : devicePixelRatio)
```

Comment:
I think this equation might be confusing because it isn't clear what pixel ratio is and why that matters to the user. The example might help more.